content/tutorials/how-to-ably-react-hooks.textile

```diff
@@ -87,47 +73,82 @@ app.get("/ably/auth", (req, res) => {
 });
 ```
 
-You can then use @configureAbly@ with an @authUrl@ so that your client will request tokens automatically:
+Next, in your React component add a reference to Ably and the @AblyProvider@:
+
+```[javascript]
+import * as Ably from 'ably'
+import { AblyProvider } from 'ably/react';
+```
+
+Then, create a new instance of an Ably client and provide that to the @AblyProvider@:
 
 ```[javascript]
-configureAbly({ authUrl: "/ably/auth" });
+const client = Ably.Realtime.Promise({ authUrl: '[YOUR_ABLY_TOKEN_API_URL_PATH]'});
+
+return (
+  <AblyProvider client={ client }>
+  </AblyProvider>
+)
 ```
 
 h2. Step 4 - Subscribe to a channel
 
-To connect and subscribe to a channel, use the @useChannel@ hook:
+To connect and subscribe to a channel, use the @useChannel@ hook within a child component of @AblyProvider@:
 
 ```[javascript]
-import { configureAbly, useChannel } from "@ably-labs/react-hooks";
+import { useChannel } from "ably/react";
 
-const [channel, ably] = useChannel("channel-name", (message) => {
-    console.log(message);
-});
+function Messages() {
+  const { channel }  = useChannel("channel-name", (message) => {
+      console.log(message);
+  });
+
+  return (<></>)
```

Comment:
I've started putting a message prompting the user to open their browser console here instead of renadering an empty value.

<div>Open your browsers console to see incoming realtime messages</div>

Or something like that.